packages/config/src/discovery/ProjectDiscovery.ts

```diff
@@ -1356,3 +1362,32 @@ function isEntryVerified(entry: ContractParameters | EoaParameters): boolean {
 
   return true
 }
+
+function allUnique(arr: string[]): boolean {
+  return new Set(arr).size === arr.length
+}
+
+function linkupActorsIntoAccounts(
+  accountsToLink: ProjectPermissionedAccount[],
+  actors: ProjectPermission[],
+): ProjectPermissionedAccount[] {
+  const result: ProjectPermissionedAccount[] = []
+  const actorNameLUT: Record<string, string> = {}
+  for (const actor of actors) {
+    assert(actor.accounts.length === 1)
+    actorNameLUT[actor.accounts[0].address] = actor.name
+  }
+
+  for (const account of accountsToLink) {
+    const entry = structuredClone(account)
+
+    const actorName = actorNameLUT[account.address]
+    if (actorName !== undefined) {
+      entry.name = actorName
+      entry.url = `#${actorName}`
```

Comment:
are we sure that
a) actor names are properly formatted for urls (e.g. no spaces)
b) are unique across actors
?

Reply:
As for (b), I've added an assertion to make sure that it's true. As for (a), they are not, we have spaces but does it matter?